components/shared/exploreTabsFilterSort/ExploreTabsFilterSort.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div class="section pb-0">
+    <h1 class="container mb-5 pb-4 explore-heading">{{ $t('explore') }}</h1>
+    <div v-if="$route.query.search" class="block">
+      {{ $t('general.searchResultsText') }}
+      <span class="text__stroked is-size-3">{{ $route.query.search }}</span>
+    </div>
+    <div class="mb-5 explore-tabs container">
+      <NeoButton
+        class="btn-collection"
+        :selected="selectedTab === tabType.COLLECTION"
+        :label="$t('collections')"
+        @click.native="updateTab(tabType.COLLECTION)" />
+      <NeoButton
+        class="btn-items"
+        :selected="selectedTab === tabType.ITEMS"
+        :label="$t('items')"
+        @click.native="updateTab(tabType.ITEMS)" />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { NeoButton } from '@kodadot1/brick'
+
+enum tabType {
+  COLLECTION = 'collectibles',
+  ITEMS = 'items',
+}
+
+const { $route, $router } = useNuxtApp()
+let selectedTab = ref($route.name?.split('-')[2])
+
+const updateTab = (val) => {
+  selectedTab.value = val
+  $route.query.page = ''
+  let queryOptions: {
+    page: string
+    search?: string | (string | null)[]
+  } = {
+    page: '1',
+  }
+  if ($route.query.search) {
+    queryOptions.search = $route.query.search
+  }
+  $router.replace({
+    path: val,
+    query: queryOptions,
+  })
+}
+</script>
+
+<style lang="scss">
+.explore-heading {
+  font-weight: 700;
+  font-size: 39px;
```

Comment:
Isnt it better to use `em` / `rem` ? cc @roiLeo ?

Reply:
Yup! or use bulma [typography helpers](https://bulma.io/documentation/helpers/typography-helpers/)

by the way I would remove some class `container mb-5 pb-4 explore-heading` and use `title`
Imo this `ExploreTabsFilterSort` need some improvement to remove fixed size